### PR TITLE
WIP - Improve Information Architecture for Helm Deployment Docs

### PR DESCRIPTION
**Pull request by briemcnally**
## Test plan

`sg run docsite`

Random Notes: https://docs.sourcegraph.com/@main/admin/deploy/kubernetes/scale#tuning-replica-counts-for-horizontal-scalability

### CODE REVIEW

File: doc/admin/deploy/kubernetes/configure.md [hunk 1]
```diff
@@ -1,11 +1,12 @@
 # Configure Sourcegraph with Kubernetes
 
-> WARNING: This guide applies exclusively to a Kubernetes deployment **without** Helm. If using Helm, please go to [Configuration in the Helm guide](helm.md#configuration). 
-> If you have not deployed Sourcegraph yet, it is higly recommended to use Helm as it simplifies the configuration and greatly simplifies the later upgrade process. See our [Helm guide](helm.md) for more information.
+**TODO: confirm these steps are required for Helm or non-helm correct ie. ingress, storage class etc. AFAIK this isn't listed on the other Helm docs**
```
Comment (kevinwojo) [inline doc/admin/deploy/kubernetes/configure.md:3]: This page is exclusively for the Kustomize workflow. With Kustomize, we have the user explicitly create their own storageClass and ingress.
 
The helm chart creates the storageClass by default. The helm chart requires the user to configure their own ingress.

File: doc/admin/deploy/kubernetes/configure.md [hunk 2]
```diff
@@ -31,95 +32,95 @@ of your deployment environment.
...
-- The sourcegraph StorageClass also allows the persistent volumes to expand their storage capacity by increasing the
+3. The sourcegraph StorageClass also allows the persistent volumes to expand their storage capacity by increasing the
  size of the related persistent volume claim.
 
-- This cannot be changed once the storage class has been created. Persistent volumes not created with the reclaimPolicy set to `Retain` can be patched with the following command:
+4. This cannot be changed once the storage class has been created. Persistent volumes not created with the reclaimPolicy set to `Retain` can be patched with the following command:
 
-```bash
+```sh
 kubectl patch pv <your-pv-name> -p '{"spec":{"persistentVolumeReclaimPolicy":"Retain"}}'
```
Comment (kevinwojo) [inline doc/admin/deploy/kubernetes/configure.md:114]: @briemcnally you need `kubectl` to create and modify the storageClass.

**Review by kevinwojo — COMMENTED** [review]
Looks like some good first pass!

File: doc/admin/deploy/kubernetes/configure.md (hunk 1 above)
Comment (briemcnally) [reply doc/admin/deploy/kubernetes/configure.md:3]: @kevinwojo is this information we should be explicitly calling out? Ie. Helm OOB configure storage classes, ingress, network config etc.? Is there any config on this page that is general k8 applicable thus applicable to helm and non-helm?

**Comment by eseliger** [comment]
As part of #wg-delete-week I am going to close this PR. If it’s still needed, feel free to reopen and restore the branch.

**Pull request closed without merging**